LandSandBoat/server
# [RUN] [sql] Update Rune Enhancement & Battuta effect flags

**_I affirm:_**

## What does this pull request do?

Verified on retail:

* All Rune Enhancements: add Logout/Zone/No Cancel/Suppress Loss Message flags
* Battuta: Add Logout/Zone flags

## Steps to test these changes


Use a rune, try to cancel it, use battuta, zone or logout and note those effects disappeared
Use a rune, wait 5 minutes, it should disappear with no log message.
